.codacy.yml

```diff
@@ -2,5 +2,7 @@ engines:
   tslint:
     enabled: true
     base_sub_dir: ui
-exclude_paths:	
+exclude_paths:
   - ui/karma.conf.js
+  - e2e/resources/python-code/**
+  - e2e/resources/r-code/**
```

Comment:
Adding these because they are effectively static test resources, which are executed in a notebook context and don't necessarily meet style guidelines, or even execute successfully (depending on what the test is looking for)